Hmm, whatever happened to the Nmake CI? I don't remember any recent changes to that, so maybe Github Actions changed, again..
```
Run call "C:\Program Files (x8[6](https://github.com/zlib-ng/zlib-ng/runs/5095292083?check_suite_focus=true#step:3:6))\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x[8](https://github.com/zlib-ng/zlib-ng/runs/5095292083?check_suite_focus=true#step:3:8)6
The system cannot find the path specified.
'nmake' is not recognized as an internal or external command,
operable program or batch file.
Error: Process completed with exit code 1.
```

--- a/zlib-ng.h.in
+++ b/zlib-ng.h.in
@@ -33,6 +33,7 @@
 #  error Include zlib-ng.h for zlib-ng API or zlib.h for zlib-compat API but not both
 #endif
 
+#ifndef RC_INVOKED
 #include "zconf-ng.h"
 
 #include <stdint.h>
@@ -40,6 +41,7 @@
 #ifndef ZCONFNG_H
 #  error Missing zconf-ng.h add binary output directory to include directories
 #endif
+#endif  /* RC_INVOKED */
 
 #ifdef __cplusplus
 extern "C" {

--- a/zlib.h.in
+++ b/zlib.h.in
@@ -34,6 +34,7 @@
 #  error Include zlib-ng.h for zlib-ng API or zlib.h for zlib-compat API but not both
 #endif
 
+#ifndef RC_INVOKED
 #include "zconf.h"
 
 #include <stdint.h>
@@ -42,6 +43,7 @@
 #ifndef ZCONF_H
 #  error Missing zconf.h add binary output directory to include directories
 #endif
+#endif  /* RC_INVOKED */
 
 #ifdef __cplusplus
 extern "C" {

